pegasus/sites.v3/code.org/views/ai_videos_link.partial

```diff
@@ -0,0 +1 @@
+<a id="ai-videos"></a>
```

Comment:
I believe this (and `ai_clear_div.partial` and `ai_20px_margin.partial` above) should be a `.html`; files only need to be `.partial` if they are _using_ the partial syntax, not if they are being included as one

Reply:
ooh, ok, will do! Thanks for the clarification